src/cmd/lium.bash

```diff
@@ -161,6 +165,9 @@ _lium() { # command current prev
   elif [ x"$prev" = x"--board" ]; then
     local BOARDS=`_lium_get_boards`
     COMPREPLY=($(compgen -W "${BOARDS}" -- $cur))
+  elif [ x"$prev" = x"--branch"]
```

Comment:
```
lium.bash: line 171: syntax error near unexpected token `elif'
lium.bash: line 171: `  elif _lium_arg_included ${prev} ${servo_serial_opts}; then'
```
It looks like `; then` at the end of this line is missing...!

Reply:
Thank you for letting me know, I fixed it!